web/src/grpc/integrateSegments.ts

```diff
@@ -25,8 +31,14 @@ export const integrateSegments = (session: Session, ctx: PeaksContext) => {
       sessionId: session.iD,
       segmentId: segmentId,
       segment: {
-        timeStart: { seconds: Math.floor(segment.startTime / 1000), nanos: 0 },
-        timeEnd: { seconds: Math.floor(segment.endTime / 1000), nanos: 0 },
+        timeStart: {
```

Comment:
Not sure why the protocols changed to string and not really sure what the string representation is...